tox.ini

```diff
@@ -32,10 +29,8 @@ commands = sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
 [testenv]
 deps =
     coverage
-    django111: Django>=1.11,<1.12
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
     django22: Django>=2.2,<2.3
+    django30: Django>=3.0,<3.1
```

Comment:
Should there be an entry for `django31` here?

Reply:
Yes, however I focused this pr on removing support for old versions rather than adding new. Adding support for 3.1 will be much easier as a second step without needing Django 1.1 support as well. (e.g there will be a number of changes re `URL()`)

Reply:
Ah. I see it's already in Travis. Let me add it.